src/data_parser.py

```diff
@@ -83,12 +84,16 @@ def get_course_details(course_id: str, term: int, year: int, offer: int, max_ret
                 and details["COURSE_TITLE"]
                 and details["TERM_DESCR"]
             ):
-                # TODO: Logger
-                # print(f"Course Details Not Found")
+                logger.error(
+                    f"Course Details Not Found\n"
+                    f"Catalog Number: {details['CATALOG_NBR']}\n"
+                    f"Course Title: {details['COURSE_TITLE']}\n"
+                    f"Term Description: {details['TERM_DESCR']}\n"
```

Comment:
Indent this by 2-4 spaces so it's clearly under the "course details not found" error

Reply:
ruff format does not allow this indent. It will default to what is above

Reply:
You can use the tab character `\t` which will do the same thing. For example, `f"\tCatalog Number: {['CATALOG_NBR']}\n"`.

Reply:
By indent, I mean inside the string, not the line itself. so rather than

```py
f"Course Details Not Found\n"
    f"Catalog Number: {details['CATALOG_NBR']}\n"
    f"Course Title: {details['COURSE_TITLE']}\n"
    f"Term Description: {details['TERM_DESCR']}\n"
````

what I mean is:

```py
f"Course Details Not Found\n"
f"  Catalog Number: {details['CATALOG_NBR']}\n"
f"  Course Title: {details['COURSE_TITLE']}\n"
f"  Term Description: {details['TERM_DESCR']}\n"
````